src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionView.tsx

```diff
@@ -26,39 +28,54 @@ const ProposedSolutionView: React.FC<ProposedSolutionViewProps> = ({
   crReviewed
 }) => {
   const faded = crReviewed != null && proposedSolution.approved === false;
+  const theme = useTheme();
 
   return (
-    <PageBlock title="" style={{ opacity: faded ? 0.5 : 1 }}>
-      {showDeleteButton && onDelete !== undefined ? (
-        <Button
-          color="error"
-          variant="outlined"
-          onClick={() => {
-            onDelete(proposedSolution);
-          }}
-          sx={{ position: 'absolute', right: 75 }}
-        >
-          <DeleteIcon />
-        </Button>
-      ) : null}
-      <Grid container rowSpacing={1} columnSpacing={{ xs: 1, sm: 2, md: 3 }}>
-        <Grid item xs={7}>
+    <Grid
+      style={{ opacity: faded ? 0.5 : 1 }}
+      sx={{
+        minHeight: 300,
+        p: 2,
+        mb: 5,
+        bgcolor: theme.palette.background.paper,
+        width: '100%',
+        borderRadius: '8px 8px 8px 8px',
+        boxShadow: 1
+      }}
+    >
+      <Grid container rowSpacing={1}>
+        <Grid item xs={12} sx={{ p: 2, mt: 1, minHeight: 100, borderRadius: 1, bgcolor: '#474848' }}>
           <DetailDisplay label="Description" content={proposedSolution.description} />
         </Grid>
-        <Grid item xs={3}>
-          <DetailDisplay label="Budget Impact" content={dollarsPipe(proposedSolution.budgetImpact)} />
-        </Grid>
-        <Grid item xs={2}>
-          {proposedSolution.approved ? <Chip label="Approved" color="success" /> : null}
-        </Grid>
-        <Grid item xs={7}>
+        <Grid item xs={12} sx={{ p: 2, mt: 2, minHeight: 100, borderRadius: 1, bgcolor: '#474848' }}>
           <DetailDisplay label="Scope Impact" content={proposedSolution.scopeImpact} />
         </Grid>
-        <Grid item xs={5}>
-          <DetailDisplay label="Timeline Impact" content={weeksPipe(proposedSolution.timelineImpact)} />
+        <Grid item xs={7} display="flex" sx={{ marginTop: 0.5 }}>
+          <AttachMoneyIcon sx={{ mr: 1 }} />
+          <Typography>{proposedSolution.budgetImpact}</Typography>
+        </Grid>
+        <Grid item xs={12} display="flex" justifyContent="space-between">
+          <Grid display="flex" sx={{ marginTop: 0.5 }}>
+            <ScheduleIcon sx={{ mr: 1 }} />
+            <Typography>{weeksPipe(proposedSolution.timelineImpact)}</Typography>
+          </Grid>
+          <Grid item>{proposedSolution.approved && <Chip label="Approved" color="success" />}</Grid>
+          {showDeleteButton && onDelete !== undefined ? (
+            <Grid item>
+              <Button
+                color="error"
+                variant="outlined"
+                onClick={() => {
+                  onDelete(proposedSolution);
+                }}
+              >
+                <DeleteIcon />
+              </Button>
+            </Grid>
+          ) : null}
```

Comment:
right here remove the : null and replace the ? with &&. This is replacing a ternary operator with an and check that will return the component if the stuff before the && operator is true